docs/result.md

```diff
@@ -1,14 +1,209 @@
-#Return_result
+# Return_result
 
 a POST request is invoked by the client to the server. It returns a JSON file with information like client_id, job, data, and jobs.
 
 ## JSON definition
 
 A job is communicated as a simple JSON object where key names and values are lower case:
 
-* `client_id`: a unique identifier of the client returning the result.
-* `job`: A job object, as defined in job.md. This job corresponds to the result. 
-* `data`: data collected from regulations.gov.
-	* how data is transferred has not been defined. 
-* `jobs`: list of job objects, as defined in job.md.
-	* list of new jobs identified with data. 
+* `client_id`: A unique identifier of the client returning the result.
+* `job_id`: ID of the current job object.
+* `data`: A path describing where the raw JSON file recieved from querying regulations, as well as the raw JSON data.
+* `jobs`: A list of new job objects. These job objects omit the job_id field. Jobs are retrieved through the JSON data collected from the regulations server.
+	* Documents have a list of docket and document jobs
+  * Document can have a list of download jobs
+  * Docket and downloads will not return any new jobs
+
+## Data Storage
+
+```
+Data (folder)
+  Agency (folder)
+    DocketID (folder)
+      docket.json
+      DocumentID (folder)
+        basic_document.json
+        document.json
+```     
+
+## Example
+
+* Docket
+```
+{
+  'client_id': '123ABC',
+  'job_id': 'ABC123',
+  'data': {
+    'agency_folder': {
+      'name': 'EPA'
+      'docket_folder': {
+        'name': 'EPA-HQ-OAR-2011-0028'
+        'file': {
+          'name': 'docket.json'
+          'json': {
+            'agency': 'EPA',
+            'docket_id': 'EPA-HQ-OAR-2011-0028'
+            'file_contents' = {
+              "agency": "Environmental Protection Agency",
+              "agencyAcronym": "EPA",
+              "cfrCitation": "40 CFR 98",
+            
+              ...
+
+              "internationalImpacts": {
+              "label": "International Impacts",
+              "value": "No"
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
```

Comment:
Too much nesting here, and this encoding forces the server to know things about the Reguations.gov process.

It should be:

```
'data': {
  'folder_name': 'EPA/EPA-HQ-OAR-2011-0028'
  'filename': 'docket.json'
  'data': { 
    <Regulations.gov data> 
  }
}

Reply:
Before I resolve this, should the folder_name and file_name fields be inside data or can they rest outside of on the same level as data? Because if we move it to the same level as data we can remove the second data and prevent from using the name data as a key twice.